### PR TITLE
Bump k8s.io dependencies to v1.19

### PR DESCRIPTION
**Pull request by swetharepakula**

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -54,46 +56,49 @@ github.com/google/go-cmp/cmp/internal/value
 github.com/google/gofuzz
 # github.com/googleapis/gax-go/v2 v2.0.5
 github.com/googleapis/gax-go/v2
-# github.com/googleapis/gnostic v0.2.0
-github.com/googleapis/gnostic/OpenAPIv2
+# github.com/googleapis/gnostic v0.4.1
 github.com/googleapis/gnostic/compiler
 github.com/googleapis/gnostic/extensions
+github.com/googleapis/gnostic/openapiv2
```
Comment (liggitt) [inline vendor/modules.txt:62]: gnostic made a breaking change to package capitalization, and openapiv2 doesn't match the capitalization of the vendor dir... you might need to delete the vendor dir and rerun `go mod vendor`

File: vendor/github.com/googleapis/gnostic/OpenAPIv2/OpenAPIv2.go [hunk 2]
```diff
@@ -7105,15 +7105,15 @@ func (m *Any) ToRawInfo() interface{} {
 // ToRawInfo returns a description of ApiKeySecurity suitable for JSON or YAML export.
```
Comment (liggitt) [inline vendor/github.com/googleapis/gnostic/OpenAPIv2/OpenAPIv2.go:7105]: this dir might need to be recreated/revendored, the capitalization does not match

**Comment by liggitt** [comment]
lgtm other than the gnostic dir

File: vendor/modules.txt (hunk 1 above)
Comment (swetharepakula) [reply vendor/modules.txt:62]: Thanks for catching that!

**Comment by freehan** [comment]
do you mind split the autogen into a separate commit?

**Comment by swetharepakula** [comment]
In the newest patch, I separated the code & go.mod changes from the vendor dir changes.

**Pull request merged**